src/components/card-search.js

```diff
@@ -17,28 +17,23 @@ export class CardSearch {
   activate(modelData) {
     if (modelData) {
       this.repo = modelData;
-
-      if(this.repo.language) {
-        let url = `/img/language-icons/${this.repo.language.toLowerCase()}.svg`;
+      if(this.repo.sourceData.language) {
+        let url = `/img/language-icons/${this.repo.sourceData.language.toLowerCase()}.svg`;
         url = url.replace(/\#/g,'_sharp');
         this.validateImage(url, (isValid) => {
           this.language_image = isValid ? url : this.language_image;
         });
       }
 
       if (modelData.releases) {
-        this.releases = modelData.releases;
-        if (!Array.isArray(modelData.releases)) {
```

Comment:
Is it ok that this if statement is gone?

Reply:
The delete is ok because in the previous version there was a conflict in the data where the "releases" where in some cases a number and in other an array, the new schema ensures that the "releases" is an array, however the "if is wrong" and should be fixed in next push